msteams-platform/build-your-first-app/build-and-run.md

```diff
@@ -1,80 +1,199 @@
 
 ## 3. Build and run your app
 
-Build and run your app locally to save time. This information is also available in the toolkit `README`. Build and run your app using the following steps:
+Build and run your app locally to save time. 
+
+**To build and run your app**
 
-1. In a terminal, go to the root directory of your app project and run `npm install`.
+1. In Visual Studio Code, select **View** > **Terminal**.
+1. Run `npm install`.
 1. Run `npm start`.
+  
+  A **Compiled successfully!** message appears in the terminal. Your app is now running on your localhost at `https://localhost:3000`. 
 
-Once complete, there's a **Compiled successfully!** message in the terminal. Your app is running on `https://localhost:3000`.
+This information is also available in the `README` file of the toolkit.
 
 ## 4. Sideload your app in Teams
 
-Your app is ready to test in Teams. To do this, you must have a Microsoft 365 development account that allows app sideloading. For more information on account opening, see [Teams development account](../build-your-first-app/build-first-app-overview.md#set-up-your-development-account). 
 
-> [!TIP]
-> Check for issues before sideloading your app, using the [validation feature in App Studio](../concepts/deploy-and-publish/appsource/prepare/submission-checklist.md#teams-app-validation-tool), which is included in the toolkit. Fix the errors to successfully sideload the app.
+Sideloading an app is the process of installing apps on your computer, that hasn't gone through the certification process, to appear in the app store and to run on a Windows device. 
 
-Sideload your app in Teams using the following steps:
+The operating system by default blocks you from sideloading apps. You have to enable sideloading of the apps to test your app in Teams. Before you turn on custom app sideloading for your development tenant, you must have one of the following:
 
-> [!NOTE]
-> To enable sideloading before you sideload your app in Teams, follow the steps in [Turn on app sideloading](../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading).
+* Ensure to have a Teams account and with admin access.
+* Ensure to have a Microsoft 365 development account that allows app sideloading. 
 
-1. Select the **F5** key to launch a Teams web client in Visual Studio Code.
-1. To display your app content in Teams, specify that where your app is running (`localhost`) is trustworthy:
-   1. Open a new tab in the same browser window (Google Chrome by default) which opened after pressing **F5**.
-   1. Go to `https://localhost:3000/tab` and proceed to the page.
-1. Go back to Teams. In the dialog, select **Add for me** to install your app.
-:::image type="content" source="../assets/images/build-your-first-app/tab-running.png" alt-text="Screenshot showing an example 'Hello, World!' personal tab app running in Teams.":::
+**To enable sideloading of your app in Teams**
```

Comment:
```suggestion
**To enable app sideloading in Teams**
```
Everything related to "enable sideloading" I think makes more sense under the "Set up your Teams development tenant" > "Do not have a tenant". This seems more like setup. By the time you get to sideloading your app, you shouldn't be doing this part.